src/application.vala

```diff
@@ -274,7 +274,9 @@ namespace Peek {
 
     private void load_stylesheets () {
       load_stylesheet_from_uri ("resource:///com/uploadedlobster/peek/css/peek.css");
-
+      if ( DesktopIntegration.get_theme_name() == "Ambiance" ){
```

Comment:
There should be no whitespace after `(` and before `)`. There *should* be a space before `{`. See the following block.